util/reflect.go

```diff
@@ -266,6 +266,14 @@ func InsertIntoStruct(parentStruct interface{}, fieldName string, fieldValue int
 		v = nv
 	}
 
+	// YANG binary fields are represented as a derived []byte value defined in the
+	// generated code. Here we cast the value to the type in the generated code.
+	if ft.Type.Kind() == reflect.Slice && t.Kind() == reflect.Slice && ft.Type.Elem().Kind() == reflect.Uint8 && t.Elem().Kind() == reflect.Uint8 {
```

Comment:
It's worth adding a comment here that this will actually trigger for slices of `uint8` -- i.e., `leaf-list` nodes of type `uint8`. We can safely treat the two as the same though since `[]byte` and `[]uint8` are equivalent in the underlying implementation (see https://play.golang.org/p/hpD9IeYY9hH for a quick check that `SetBytes` won't panic).

Reply:
Added comment for `[]uint8` behavior

Reply:
Great, thanks.